src/OpenTelemetry.Contrib.Instrumentation.Quartz/README.md

```diff
@@ -0,0 +1,98 @@
+# QuartzNET Instrumentation for OpenTelemetry .NET
+
+Automatically instruments the Quartz jobs from
+[Quartz](https://www.nuget.org/packages/Quartz/).
+
+## Installation
+
+```shell
+dotnet add package OpenTelemetry.Extensions.Hosting
+dotnet add package OpenTelemetry.Contrib.Instrumentation.Quartz
+```
+
+## Configuration
+
+ASP.NET Core instrumentation example:
+
+```csharp
+// Add QuartzNET inside ConfigureServices
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddQuartz(q =>
+    {
+        // base quartz scheduler, job and trigger configuration
+    });
+
+    // ASP.NET Core hosting
+    services.AddQuartzServer(options =>
+    {
+        // when shutting down we want jobs to complete gracefully
+        options.WaitForJobsToComplete = true;
+    });
+}
+
+// Add OpenTelemetry and Quartz instrumentation
+services.AddOpenTelemetryTracing(x =>
+{
+    x.AddQuartzInstrumentation();
+    x.UseJaegerExporter(config => {
+      // Configure Jaeger       
+    });
+});
+```
+
+## Filter traced operations
+
+For example you can trace only consume and handle operations using this snippet:
+
+```csharp
+// ...
+using OpenTelemetry.Instrumentation.Quartz.Implementation;
+// ...
+x.AddQuartzInstrumentation(
+    opts =>
+        opts.TracedOperations = new HashSet<string>(new[] {
+            OperationName.Job.Execute,
```

Comment:
would be nice to provide a list of supported, or is this free form and can be anything?

Reply:
updated to succinct supported list